static/scripts/pay.ts

```diff
@@ -5,14 +5,86 @@ import { TxType, txData } from "./render-transaction";
 
 const permit2Address = "0x000000000022D473030F116dDEE9F6B43aC78BA3";
 
+const notifications = document.querySelector(".notifications") as Element;
+const claimButtonElem = document.getElementById("claimButton") as Element;
+const buttonMark = document.querySelector('.hide-load');
+const claimLoader = document.querySelector(".claim-loader") as Element;
+
+// Object containing details for different types of toasts
+const toastDetails = {
+    timer: 5000,
+    success: {
+        icon: 'fa-circle-check',
+    },
+    error: {
+        icon: 'fa-circle-xmark',
+    },
+    warning: {
+        icon: 'fa-triangle-exclamation',
+    },
+    info: {
+        icon: 'fa-circle-info',
+    }
+}
+
+const removeToast = (toast) => {
+    toast.classList.add("hide");
+    if(toast.timeoutId) clearTimeout(toast.timeoutId); // Clearing the timeout for the toast
```

Comment:
use {curly-braces} for better readability

Reply:
Didn't get that, do you mean passing the parameters as object or a template literal for adding classes

Reply:
`if (condition) {...}`

Reply:
Oh okay, would change.. Used a one-liner there

Reply:
> use {curly-braces} for better readability

We should enforce all this in our linter @rndquu